src/main/java/com/amazonaws/services/kinesis/multilang/MultiLangDaemonConfig.java

```diff
@@ -107,6 +106,50 @@ public MultiLangDaemonConfig(String propertiesFile,
         prepare(processingLanguage);
     }
 
+    private ClientConfiguration buildClientConfig(Properties properties)
+    {
```

Comment:
Could you use the [formatter](https://github.com/awslabs/amazon-kinesis-client/tree/master/formatter) provided with the new changes.